style.css

```diff
@@ -1,3 +1,104 @@
-#root {
-  color: red;
+body {
+  background-color: rgb(218, 218, 218);
+  text-align: center;
+  font-size: 20px;
+  font-family: "Sansita Swashed", cursive;
+  margin: 0;
+}
+
+.header {
+  text-align: left;
+  grid-column: 1;
+  margin-bottom: 2.5rem;
+  background-color: #0072ce;
+  color: #fff;
+  margin: 0;
+  padding: 2em;
+}
+
+.episode-dropdown {
+  display: none;
+}
+
+.episode-search {
+  display: none;
+}
+
+.show {
+  border: 1px solid #000;
+  margin-bottom: 2em;
+  padding: 2em;
+}
+
+.episode {
+  background-color: #fff;
+  border-radius: 15px;
+}
+
+.episode-title-container {
+  border: 1px solid #000;
+  border-radius: 15px;
+}
+
+.episode-title {
+  font-weight: 600;
+  font-size: 1.5em;
+}
+
+.episode-thumbnail-container {
+  margin: 2rem 5rem 0;
+}
+
+.episode-description {
+  color: rgb(97, 97, 97);
+  margin: 1rem 3rem;
+}
+
+@media screen and (min-width: 900px) {
+  .main {
+    margin: 2.5em;
+  }
+
+  .episode-search,
+  .show-search {
+    margin-left: 1.5rem;
+  }
+
+  .search-stats {
+    display: none;
+  }
+
+  .all-episodes {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    grid-gap: 5rem;
+  }
+
+  .show {
+    display: grid;
+    grid-template-columns: auto 2fr 25rem;
+    grid-template-rows: repeat(2, auto);
+    grid-gap: 2em;
+    text-align: left;
+  }
+
+  .show-title {
+    grid-column: 1 / 4;
+  }
+
+  .show-thumbnail {
+    grid-row: 2;
+  }
+
+  .show-stats {
+    border: 1px solid #000;
+    grid-column: 3;
+    padding: 2em;
+  }
+}
+
+@media screen and (max-width: 600px) {
+  .show-search {
+    margin-left: 0;
+  }
```

Comment:
Your design is responsive and the page renders well on different screen sizes. Perhaps you could make input boxes a bit bigger and add some padding to the body  I really like the font and your choice of colour palette for the page that make the page simplistic yet stylish.